force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,28 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
+
+        String str = 'Hello, Salesforce!';
         
-        // Get the length of the String        
+        // Get the length of the String    
+        Integer length = str.length();
         // Get a substring of the String of the first 5 characters
+        String sub = str.substring(0, 5);
+
         // Convert the String to lower case
-        
+
+        String lowerCase = str.toLowerCase();
         // Return the list with the string data
         // In the first position add the string length
         // In the second position add the substring
         // In the third position add the string in lower case
 
-        return null; // Replace null with the variable you used to store the result
+List<String> result = new List<String>();
+        result.add(String.valueOf(length));
+        result.add(sub);
+        result.add(lowerCase);
+
+        return result; // Replace null with the variable you used to store the result
```

Comment:
Something I encourage students to think about is, "Do I really need a variable here?" In this case, I would encourage you to add these values directly to the List. It takes half as much code and I don't think it hurts readability at all. 

```
 public static List<String> practiceStringMethods() {
        List<String> strings = new List<String>();
        String helloSalesforce = 'Hello, Salesforce!';
        strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
    }
 ```